### DISCUSSION TITLE
New pro version of spotlight easy to be included with Filament?

### DISCUSSION THREAD

**Discussion by vv-ac** [wire-elements/spotlight#112] [General]
Hi there!
I've seen that spotlight is part of wire elements pro which is fair enough; it's a cool feature.
Does anyone know if it's a long way to include the pro version into Filament (filamentphp) to make it acknowledge the resources etc.?

There is a package to include the "old/free" version of spotlight into it but I do not know about the paid one.

Thanks in advance, have a pleasant Thursday.
vv-ac

**Comment by vv-ac** [comment]
Still interested!
Ofc, it would be nice to have it in v3 of filament.

**Reply by PhiloNL** [reply]
Thanks for letting me know :)
I might look into this once everything is compatible with Livewire v3 and things are more stable.

**Reply by vv-ac** [reply]
Cool, thanks. The first stable release of filament v3 has been released, livewire though still seems to be in a beta mode.

**Reply by vv-ac** [reply]
Hey there. Time is flying, happy December!
Any plans yet to migrate Spotlight?

Merry Christmas soon :).
